packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/Unbranded-TypeSpec.tsp

```diff
@@ -410,3 +410,13 @@ op ListWithContinuationTokenHeaderResponse(@query @continuationToken token?: str
 
   @header @continuationToken nextToken?: string;
 };
+
+model Page<T> {
```

Comment:
We should consider unit test only vs adding more stuff to the sample.  I think the current sample project is already too large but will defer if you really want this in here.

In either case can we add the unit tests?